pkg/config/configuration.go

```diff
@@ -53,9 +53,7 @@ type SelectorField struct {
 }
 
 type TracingSpec struct {
-	Enabled      bool `json:"enabled" yaml:"enabled"`
-	ExpandParams bool `json:"expandParams" yaml:"expandParams"`
-	IncludeBody  bool `json:"includeBody" yaml:"includeBody"`
+	Enabled bool `json:"enabled" yaml:"enabled"`
```

Comment:
will you switch to `samplingRate` later?

Reply:
I will change it in a separate PR , tracking issue #1395